index.html

```diff
@@ -62,15 +70,14 @@
 
 * [FedCM](https://github.com/w3ctag/design-reviews/issues/718). Third-party cookies have been used to support single sign-on. FedCM is a set of technologies built to support identity federation, without replicating all functionality of third-party cookies.
 * [CHIPS](https://github.com/w3ctag/design-reviews/issues/654). The goal of CHIPS is to allow state, without supporting correlation or tracking between web sites which don't know they are collaborating, for example, when embedding third-party services like customer service webchat.
```

Comment:
People keep saying that CHIPS is a replacement worthy of mention here.  It is not.

Yes, that leaves us with one example.  That's OK.  I suggest that we restructure around a single example instead.

Reply:
I'm fine with keeping or dropping the CHIPS example, but I'd love to hear more about why you want to drop it.

Reply:
I'd really prefer to have at least 2 examples of what we're talking about. Do you strongly object to listing CHIPS? Would you be ok with listing CHIPS but hedging the language a bit?